docs/relationships.md

```diff
@@ -110,6 +110,25 @@ const book = {
 }
 ```
 
+### Metadata
+
+It is possible to assign metadatas to relationships. You need to modify the content of the document relationships somehow and save the modified document.
+
+```javascript
+const doc = {
+  _id: "mobydick",
+  relationships: {
+    authors: {
+      data: [{
+        _id: "hermanmelville",
+        _type: "io.cozy.contacts"
+      }]
+    }
+  }
+}
+doc.relationships.authors.data[0].metadata = { addressId: "123" }
+await client.save(doc)
```

Comment:
This seems kind of tedious :/ Using `doc.relationships.authors.add({_id: ..., metadata: ...})` does not work?

Reply:
@paultranvan non ça ne fonctionne pas, l'argument est le doc en question à lier, alors que concernant les metadata on ne veut pas les mettre sur le doc mais sur la relationships elle-même.

Reply:
Ok, let's make an issue then: https://github.com/cozy/cozy-client/issues/1393